hhs_oauth_server/request_logging.py

```diff
@@ -97,10 +97,10 @@ def __str__(self):
         # Get FHIR_ID if available.
         user = get_user_from_request(self.request)
         if user:
-            log_msg['user'] = str(user)
```

Comment:
Just want to check if this field is still needed (I don't see it defined anywhere else).